src/components/ListOfGoods.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const getGoods = (goods, isReverse, sortByAlphabet, SortByLength) => {
+  const newGoods = [...goods];
+
+  if (sortByAlphabet) {
+    newGoods.sort((g1, g2) => g1.localeCompare(g2));
+  }
+
+  if (SortByLength) {
+    newGoods.sort((g1, g2) => g1.length - g2.length);
+  }
+
+  if (isReverse) {
+    newGoods.reverse();
+  }
+
+  return newGoods;
+};
+
+class ListOfGoods extends React.Component {
+  state = {
+    isReverse: false,
+    sortByAlphabet: false,
+    SortByLength: false,
```

Comment:
you can create one sortBy field and save string there, witch represent by what condition you want to sort